hack/load-testing.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# This script is used to load test Cincinnati instance.
+# It uses vegeta - `go get -u github.com/tsenart/vegeta`
+
+# PE_URL=$(oc -n cincinnati-e2e get route cincinnati-policy-engine -o jsonpath='{.spec.host}')
+# export GRAPH_URL="http://${PE_URL}/api/upgrades_info/v1/graph"
+
+
+mkdir /tmp/results
```

Comment:
Should we use a directory created by `mktemp -d` instead? Otherwise subsequent runs could clobber the previous ones.